Makefile

```diff
@@ -75,23 +72,15 @@ build: dist build-dirs Makefile
 	ARCH=$(ARCH) OS=$(OS) VERSION=$(VERSION) PKG=$(PKG) ./hack/build.sh
 	cp $(BINARIES) ./bin/$(OS)_$(ARCH)/
 
-# Note: Only used for development, i.e. in CI the images are built using Wercker.
-.PHONY: containers
-containers: $(CONTAINER_FILES)
-.container-$(REGISTRY_STRING)-%-$(VERSION): build dist
-	@echo Builing container: $*
-	@docker login -u '$(DOCKER_REGISTRY_USERNAME)' -p '$(DOCKER_REGISTRY_PASSWORD)' $(REGISTRY)
-	@docker build --build-arg=http_proxy --build-arg=https_proxy -t $(REGISTRY)/$*:$(VERSION) -f docker/$*/Dockerfile .
-	@docker images -q $(REGISTRY)/$*:$(VERSION) > $@
-
 # Note: Only used for development, i.e. in CI the images are pushed using Wercker.
 .PHONY: push
-push: $(PUSH_FILES)
-.push-$(REGISTRY_STRING)-%-$(VERSION): .container-$(REGISTRY_STRING)-%-$(VERSION)
-	@echo Pushing container: $*
-	@docker login -u '$(DOCKER_REGISTRY_USERNAME)' -p '$(DOCKER_REGISTRY_PASSWORD)' $(REGISTRY)
-	@docker push $(REGISTRY)/$*:$(VERSION)
-	@docker images -q $(REGISTRY)/$*:$(VERSION) > $@
+push: build
+	@docker build --build-arg=http_proxy --build-arg=https_proxy -t $(REGISTRY)/$(TENANT)/mysql-operator:$(VERSION) -f docker/mysql-operator/Dockerfile .
```

Comment:
Would break these up over multiple lines

Reply:
Agree. Split into a separate step for hygiene and broken into multiple lines.